src/js/features/search/makeGetKatsuPublic.thunk.ts

```diff
@@ -1,19 +1,25 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
 import axios from 'axios';
-import { katsuUrl } from '@/constants/configConstants';
+import { katsuPublicSearchUrl } from '@/constants/configConstants';
+import type { RootState } from '@/store';
 import type { KatsuSearchResponse } from '@/types/search';
 import { printAPIError } from '@/utils/error.util';
-import type { RootState } from '@/store';
+import { authorizedRequestConfig } from '@/utils/requests';
 
 export const makeGetKatsuPublic = createAsyncThunk<
   KatsuSearchResponse,
   void,
   { state: RootState; rejectValue: string }
->('query/makeGetKatsuPublic', (_ignore, thunkAPI) => {
-  const queryParams = thunkAPI.getState().query.queryParams;
+>('query/makeGetKatsuPublic', (_, { rejectWithValue, getState }) => {
+  const state = getState();
+  const scopeParams = state.metadata.selectedScope;
+  const queryParams = { ...scopeParams, ...state.query.queryParams };
```

Comment:
Rename to something like `scopedQueryParams`, since it includes both.

Reply:
We could also modify `scopedAuthorizedRequestConfig` to have a `useQuery` boolean argument that tells the function to include `state.query.queryParams`.

Reply:
added an extra argument to `scopedAuthorizedRequestConfig` for arbitrary extra query params